pkg/remote/terraform/provider.go

```diff
@@ -10,76 +10,43 @@ import (
 	"time"
 
 	"github.com/cloudskiff/driftctl/pkg/parallel"
-	"github.com/sirupsen/logrus"
-
 	tf "github.com/cloudskiff/driftctl/pkg/terraform"
-
-	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/eapache/go-resiliency/retrier"
 	"github.com/hashicorp/terraform/plugin"
 	"github.com/hashicorp/terraform/plugin/discovery"
 	"github.com/hashicorp/terraform/providers"
 	"github.com/hashicorp/terraform/terraform"
 	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
 	"github.com/zclconf/go-cty/cty"
 	"github.com/zclconf/go-cty/cty/gocty"
 )
 
-type awsConfig struct {
-	AccessKey     string
-	SecretKey     string
-	CredsFilename string
-	Profile       string
-	Token         string
-	Region        string `cty:"region"`
-	MaxRetries    int
-
-	AssumeRoleARN         string
-	AssumeRoleExternalID  string
-	AssumeRoleSessionName string
-	AssumeRolePolicy      string
-
-	AllowedAccountIds   []string
-	ForbiddenAccountIds []string
-
-	Endpoints        map[string]string
-	IgnoreTagsConfig map[string]string
-	Insecure         bool
-
-	SkipCredsValidation     bool
-	SkipGetEC2Platforms     bool
-	SkipRegionValidation    bool
-	SkipRequestingAccountId bool
-	SkipMetadataApiCheck    bool
-	S3ForcePathStyle        bool
+type TerraformProviderConfig struct {
+	DefaultAlias      string
+	GetProviderConfig func(alias string) interface{}
```

Comment:
Maybe just above the struct an explanation of what is the alias attribute.

Something like: `Alias refers to the 'meta-region' where the terraform provider will scan all resources`

Reply:
WDYT of 

```
// "alias" in these struct is a way to namespaces gRPC client
// for example, if we need to read S3 bucket from AWS from multiples region, we'll have
// an alias per region, and the alias IS the region itself
// So we can query resources using a specific custom provider configuration
```

Reply:
Nice !

Just quick typo here `"alias" in this ... to namespace ...` `from multiple AWS region, ...`